pkg/base/helm.go

```diff
@@ -300,6 +300,12 @@ func kustomizeHelmNamespace(baseFiles []BaseFile, renderOptions *RenderOptions)
 			continue // ignore invalid resources
 		}
 
+		// ignore crds
```

Comment:
what does this do? why is this in this pr?

Reply:
I had to update the kustomize dependency, which changed the functionality. This was needed for tests to pass. Agreed that this is super suspect, but reverting the Krusty update makes it impossible (for me) to mod in k8s